testing/scripts/test_bad_graphs.py

```diff
@@ -7,7 +7,7 @@
 from k8s_utils import *
 
 def wait_for_status(name):
-    for attempts in range(3):
+    for attempts in range(5):
```

Comment:
This change is unrelated. I just noticed that it can occasionally take a bit longer for the status to appear.